### PR TITLE
add missing translations

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?179685

@dannyroberts @orangejenny

### CODE REVIEW

**Comment by orangejenny** [comment]
looks good. `config.report.title` isn't translate-able?

File: corehq/apps/app_manager/app_strings.py [hunk 1]
```diff
@@ -74,13 +74,17 @@ def maybe_add_index(text):
         if audio:
             yield id_strings.module_audio_locale(module), audio
 
+        _provide_translatable = lambda string, lang: {lang: string}
+        _default_translatable = lambda string: _provide_translatable(string, "default")
+        _en_translatable = lambda  string: _provide_translatable(string, "en")
```
Comment (dimagimon) [inline corehq/apps/app_manager/app_strings.py]: E271 multiple spaces after keyword

**Comment by nickpell** [comment]
@orangejenny Good point - fixed.  Also fixed so that ```trans``` is passed a ```dict``` instead of ```str```

**Comment by orangejenny** [comment]
nice, looks good once tests pass

**Comment by nickpell** [comment]
hold off on merge, need to fix translations

**Pull request closed without merging**